packages/client/src/store/commonSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+interface CommonState {
+  currentUser: string,
+  theme: string,
+  language: string,
+}
+
+const initialState: CommonState = {
+  currentUser: 'Yolo man',
+  theme: 'default',
+  language: 'RU',
+};
+
+export const counterSlice = createSlice({
+  name: 'common',
+  initialState,
+  reducers: {
+    setUser: (state, action) => {
```

Comment:
action нужно типизировать, сейчас  тип у action.payload - any.

```suggestion
    setUser: (state, action: PayloadAction<string>) => {
```